builder/Dockerfile

```diff
@@ -65,12 +60,63 @@ USER dw
 
 ENTRYPOINT [ "/bin/bash" ]
 
+########################################
+# The golang build stage
```

Comment:
Separating the golang build stage and the artifact build stage allows us to build go code separately from the python code